usmqe_tests/api/ceph/test_ceph_cluster.py

```diff
@@ -111,8 +113,10 @@ def test_cluster_import_valid(valid_session_credentials):
 
     LOGGER.debug("integration_id: %s" % integration_id)
     pytest.check(
-        [x for x in api.get_cluster_list() if x["integration_id"] == integration_id],
-        "Job list integration_id '{}' should be present in cluster list.".format(integration_id),
+        [x for x in api.get_cluster_list()
+         if x["integration_id"] == integration_id],
+        "Job list integration_id '{}' should be "
+        "present in cluster list.".format(integration_id),
```

Comment:
I think changed code is less readable than origin. Do you know at first sight where does list end? I think this is more obvious in origin.